reports/Top_10_products_more_ordered.sql

```diff
@@ -0,0 +1,6 @@
+SELECT p.id, p."name", SUM(op.quantity) as QTD
+FROM products p
+inner join order_products op on op.product_id = p.id
+group by p.id
+order by QTD desc
+limit 10
```

Comment:
é bom evitar utilizar letras maiúsculas em variáveis sql, pra não serem confundidas com palavras reservadas

```suggestion
SELECT p.id, p."name", SUM(op.quantity) as qtd
FROM products p
inner join order_products op on op.product_id = p.id
group by p.id
order by qtd desc
limit 10
```